package.json

```diff
@@ -92,15 +94,14 @@
     "webpack-merge": "^5.7.3"
   },
   "scripts": {
-    "postinstall": "lerna bootstrap && link-parent-bin && link-parent-bin -c package-templates && yarn run tryBuild",
```

Comment:
we really need to have this `postinstall` script here, otherwise many of internal dependencies break very easily. you can track down when it was added, maybe there's a better reason why but still - very sure we should keep it